draft-ietf-tls-tls13.md

```diff
@@ -2957,8 +2956,17 @@ the responses).
 
 
 ### Key and IV Update {#key-update}
+     enum { update_not_requested(0), update_requested(1), (255)
```

Comment:
Do enums ever note, syntactically, whether they're complete or not? NamedGroup certainly isn't, but it doesn't say anything. Interestingly, CipherSuite isn't even an enum. It's a uint8[2].